master/inventory.py

```diff
@@ -88,7 +88,8 @@ def build_names(platform, arch, name):
     elif name[:7] == "freebsd":
         os_name = "freebsd"
         os_pkg_ext = "tar.gz"
-        make_cmd = "gmake USE_BINARYBUILDER_LIBUV=0"
+        make_cmd = "gmake"
+        flags += 'USE_BINARYBUILDER_LIBUV=0 ' # https://github.com/JuliaLang/julia/issues/34627
```

Comment:
This should only be necessary for FreeBSD 12 and later. For 11, I think we do want to use BinaryBuilder libuv.

Reply:
https://github.com/JuliaCI/julia-buildbot/pull/221